metron-deployment/roles/quick_dev/tasks/main.yml

```diff
@@ -15,23 +15,50 @@
 #  limitations under the License.
 #
 ---
-#
-# Workaround for Kafka not starting
-# Fire off async start followed by
-# Sync start -execution will pause until
-# final start completes.
-#
-- name: Start the ambari cluster - no wait
-  ambari_cluster_state:
+- name: Delete the Metron Components from Ambari
```

Comment:
It bridges the gap between the quick-dev image and a fully functioning instance. To wit:

- Removes the stale Metron that ships with the image from Ambari
- Removes the stale Metron RPMs
- Re-installs fresh Metron packages
- Starts the cluster

We only require these activities on a quick-dev image. If you think of Ansible as declarative, it's not that weird. We apply the quick-dev role to the quick-dev host.